docs/modules/ROOT/pages/gsn.adoc

```diff
@@ -0,0 +1,77 @@
+= The Gas Station Network
+
+User onboarding is one of the hottest topics in Ethereum, and meta-transactions the de-facto solution to this important problem. Here you'll learn all about the Gas Station Network (GSN) and how to write contracts that don't require their users to hold Ether to pay for gas.
+
+If you're already up to speed with the workings of the GSN, feel free to skip to [receiving-a-relayed-call], where we'll go over how to use the OpenZeppelin Contracts to easily write a GSNRecipient contract. Otherwise, strap in!
+
+== Sending gas-less transactions
+
+All Ethereum transactions spend gas, and the sender of the transaction must have enough Ether to pay for these gas costs. Even though these costs are low for basic transactions (a couple cents), getting this Ether is no easy task: you'll probably need to go through Know Your Customer and Anti Money-Laundering processes (KYC & AML), which not only take time but often involve sending a selfie holding your passport over the Internet (!). Only the most hardcore users will put up with this hassle, and dApp adoption greatly suffers when Ether is required. We can do better.
```

Comment:
Loved the intro! The ending of the first sentence looks odd though: it's hard to follow what _these gas costs_ refer to. Also, be careful with the change of subject: in the first sentence it's "the sender of the tx", in the second it's "you", and the third is "the most hardcore users".